<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataLogger</class>
 <widget class="QMainWindow" name="DataLogger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataLogger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="SideBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>141</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="MainBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="HomeButton">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>18</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/home.svg</normaloff>:/icons/icons/home.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="AlarmButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>18</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Alarm</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/bell.svg</normaloff>:/icons/icons/bell.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="LogoutButton">
    <property name="geometry">
     <rect>
      <x>766</x>
      <y>18</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/log-out.svg</normaloff>:/icons/icons/log-out.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Stab1_button">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>136</y>
      <width>103</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stab 1</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/radio.svg</normaloff>:/icons/icons/radio.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="FileView">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>88</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>File View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stab1_button_2">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>304</y>
      <width>103</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stab 4</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/radio.svg</normaloff>:/icons/icons/radio.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Stab1_button_3">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>248</y>
      <width>103</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stab 3</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/radio.svg</normaloff>:/icons/icons/radio.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Stab1_button_4">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>192</y>
      <width>103</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stab 2</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/radio.svg</normaloff>:/icons/icons/radio.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>196</y>
      <width>693</width>
      <height>211</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>693</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>693</width>
      <height>435</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Temperature</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Humidity</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="PaneView">
    <property name="geometry">
     <rect>
      <x>436</x>
      <y>82</y>
      <width>103</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Panel View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/chevron-left.svg</normaloff>:/icons/icons/chevron-left.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="DownlaodPDF">
    <property name="geometry">
     <rect>
      <x>726</x>
      <y>78</y>
      <width>127</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Download PDF</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/file-text.svg</normaloff>:/icons/icons/file-text.svg</iconset>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="from_dateTime">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>134</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>164</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="to_dateTime">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>162</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Search">
    <property name="geometry">
     <rect>
      <x>728</x>
      <y>152</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/search.svg</normaloff>:/icons/icons/search.svg</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>2</number>
  </property>
  <property name="gridDeltaY">
   <number>2</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
